jenkins/Jenkinsfile

```diff
@@ -0,0 +1,34 @@
+pipeline {
+
+    options {
+        disableConcurrentBuilds()
+        buildDiscarder(logRotator(numToKeepStr: '5'))
+        timeout(time: 5, unit: 'MINUTES')
+    }
+
+    agent any
+
+    triggers {
+        pollSCM('H/5 * * * *')
+    }
+
+    stages {
+        stage('Build') {
+            agent {
+                docker {
+                    reuseNode true
+                    image 'adoptopenjdk/maven-openjdk11:latest'
+                    args '-v $HOME/.m2:/root/.m2'
```

Comment:
is $HOME the workspace?  seems like there's a workspace variable we should be using here instead.